content/doc/administration/requirements/java.adoc

```diff
@@ -58,3 +58,18 @@ Modern versions of Jenkins controllers and Jenkins agents verify Java requiremen
 and notify users when they are launched with an unsupported version.
 
 The plugin:versioncolumn[Versions Node Monitors plugin] provides detailed Java version monitoring.
+
+== Java Development Kits Used
+
+Jenkins uses Eclipse Temurin as its primary JDK for building and testing Java based applications.
+This includes:
+
+* Container Images
+* Controller Builds
+* Testing Infrastructure
+
+Some of the reasons for choosing Temurin are:
+
+* It's availability over a wide range of platforms, including different operating systems and architectures, over many different Java SE versions.
```

Comment:
```suggestion
* Availability over a wide range of platforms, including different operating systems and architectures, over many different Java SE versions.
```
Otherwise, it would be `Its availability over a wide range of platforms, including different operating systems and architectures, over many different Java SE versions.`